macos/FluentUI/Colors.swift

```diff
@@ -0,0 +1,357 @@
+			case .presenceBlocked:
+				return "presenceBlocked"
+			case .presenceBusy:
+				return "presenceBusy"
+			case .presenceDnd:
+				return "presenceOffline"
+			case .presenceOffline:
+				return "presenceOffline"
+			case .presenceOof:
+				return "presenceOof"
+			case .presenceUnknown:
+				return "presenceUnknown"
+			}
+		}
+	}
+	
+	@available(*, deprecated, renamed: "textOnAccent")
+	@objc public static var foregroundOnPrimary: NSColor = textOnAccent
+	
+	// MARK: Physical - grays
+	
+	@objc public static let gray950: NSColor = Palette.gray950.color
+	@objc public static let gray900: NSColor = Palette.gray900.color
+	@objc public static let gray800: NSColor = Palette.gray800.color
+	@objc public static let gray700: NSColor = Palette.gray700.color
+	@objc public static let gray600: NSColor = Palette.gray600.color
+	@objc public static let gray500: NSColor = Palette.gray500.color
+	@objc public static let gray400: NSColor = Palette.gray400.color
+	@objc public static let gray300: NSColor = Palette.gray300.color
+	@objc public static let gray200: NSColor = Palette.gray200.color
+	@objc public static let gray100: NSColor = Palette.gray100.color
+	@objc public static let gray50: NSColor = Palette.gray50.color
+	@objc public static let gray25: NSColor = Palette.gray25.color
+	
+	// MARK: Physical - Non-grays
+	
+	@objc public static let error: NSColor = Palette.dangerPrimary.color
+	@objc public static let warning: NSColor = Palette.warningPrimary.color
+	
+	@objc public static var avatarBackgroundColors: [NSColor] = [
+		Palette.cyanBlue10.color,
+		Palette.red10.color,
+		Palette.magenta20.color,
+		Palette.green10.color,
+		Palette.magentaPink10.color,
+		Palette.cyanBlue20.color,
+		Palette.orange20.color,
+		Palette.cyan20.color,
+		Palette.orangeYellow20.color,
+		Palette.red20.color,
+		Palette.blue10.color,
+		Palette.magenta10.color,
+		Palette.gray40.color,
+		Palette.green20.color,
+		Palette.blueMagenta20.color,
+		Palette.pinkRed10.color,
+		Palette.gray30.color,
+		Palette.blueMagenta30.color,
+		Palette.gray20.color,
+		Palette.cyan30.color,
+		Palette.orange30.color
+	]
+	
+	/// Used for hyperlinks
+	@objc public static let communicationBlue: NSColor = Palette.communicationBlue.color
+	
+	// MARK: Base semantic
+	
+	@available(*, deprecated, renamed: "textDisabled")
+	@objc public static let disabled: NSColor = textDisabled
+	
+	/// text color should not be lower than `gray500` in light mode to achieve 4.5:1 minimum contrast ratio in `.white` background
+	/// text color should not be higher than `gray400` in dark mode to achieve 4.5:1 minimum contrast ratio in `.black` background
+	/// when determining high contrast color, add 200 in light mode and substract 200 in dark mode from the default color.
+	
+	/// text color used for main level in the screen. eg. title in dialog, title in navigationbar with `surfacePrimary`, etc
+	@objc public static let textDominant = NSColor(light: gray900, lightHighContrast: .black, dark: .white)
```

Comment:
adding these semantic colors seems to be bit out of scope of this PR. did you ask Rafael if these colors works for Mac?